plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/N4JSWorkspaceManager.java

```diff
@@ -24,19 +25,24 @@
 import org.eclipse.xtext.workspace.IWorkspaceConfig;
 import org.eclipse.xtext.xbase.lib.Procedures.Procedure2;
 
+import com.google.inject.Singleton;
+
 /**
  *
  */
 @SuppressWarnings("restriction")
+// TODO: Xtext - make WorkspaceManager a @Singleton?
```

Comment:
same (also some other occurrences in this file)

Reply:
This and similars also.